notebooks/en/semantic_reranking_elasticsearch.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>In this notebook, we will..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>